src/stores/A_store.js

```diff
@@ -153,6 +153,45 @@ class A_store extends ReduceStore {
                 }
             }
 
+            case ActionTypes.DSTARLITE: {
+                const {player, end, pathGrid} = state;
+                const {grid} = pathGrid;
+                const DSL = new Dstarlite(grid[player.y][player.x], grid[end.y][end.x], Cell.heuristic);
```

Comment:
direct grid access outside of `Grid` 🤔

Reply:
also, variables that are not a class and not a constant (and not a react component) should not start with a capital letter

(this is an *instance of a class*, not a class)